src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -15,8 +15,9 @@
     public static final Prefix PREFIX_START_DATE = new Prefix("s/");
     public static final Prefix PREFIX_END_DATE = new Prefix("e/");
     public static final Prefix PREFIX_DESCRIPTION = new Prefix("d/");
-    public static final Prefix PREFIX_DEPARTMENT = new Prefix("dep/");
+    public static final Prefix PREFIX_DEPARTMENT = new Prefix("dep");
```

Comment:
will this affect creating/editing persons. as we use dep/ for those commands

Reply:
yes it will affect

Reply:
actually as long the prefix is consistently used throughout the code, then it wont really affect right?

Reply:
yes but we want to use dep/ not dep for addEvent and editEvent commands.